### DISCUSSION TITLE
Microsoft Windows-like splitter support

### DISCUSSION THREAD

**Discussion by onmishkin** [rxhanson/Rectangle#679] [Ideas]
On Microsoft Windows the builtin Rectangle-like keyboard shortcut window management feature has an ability that AFAIK Rectangle doesn't have: After you make two side-by-side half-screen-width full-screen-height windows (using the Windows-Left and Windows-Right keys), you can simultaneously change the width of the two windows by dragging a splitter that appears when you hover on the touching edges of the two windows. I'm guessing that Rectangle doesn't do that and in fact it might not be _possible_ to do that but I thought I'd mention the idea anyway.

Any thoughts?

**Comment by rxhanson** [comment]
The reason this isn't in Rectangle is because resizing windows from the left side programmatically as a 3rd party developer can be really janky. If you've tried Amethyst, you might know what I mean. If someone figures out an implementation that fixes this behavior, I see no reason to reject a good pull request with it though.
